seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java

```diff
@@ -79,4 +82,76 @@ public static void createNewFile(String filePath) {
             createParentFile(file);
         }
     }
+
+    /**
+     * return the line number of file
+     *
+     * @param filePath The file need be read
+     * @return
+     */
+    public static Long getFileLineNumber(@NonNull String filePath) {
+        try {
+            return Files.lines(Paths.get(filePath)).count();
+        } catch (IOException e) {
+            throw new SeaTunnelException(String.format("get file[%s] line error", filePath), e);
+        }
+    }
+
+    /**
+     * return the line number of all files in the dirPath
+     *
+     * @param dirPath dirPath
+     * @return
+     */
+    public static Long getFileLineNumberFromDir(@NonNull String dirPath) {
+        File file = new File(dirPath);
+        Long value = null;
+        if (file.isDirectory()) {
+            value = Arrays.stream(file.listFiles()).map(currFile -> {
+                if (currFile.isDirectory()) {
+                    return getFileLineNumberFromDir(currFile.getPath());
+                } else {
+                    return getFileLineNumber(currFile.getPath());
+                }
+            }).mapToLong(Long::longValue).sum();
+        } else {
+            value = getFileLineNumber(file.getPath());
+        }
+
+        return value;
+    }
+
+    /**
+     * clear dir and the sub dir
+     *
+     * @param filePath filePath
+     * @return
+     */
+    public static void deleteFile(@NonNull String filePath) {
+        File file = new File(filePath);
+        if (file.exists()) {
+            if (file.isDirectory()) {
+                deleteFiles(file);
+            }
+            file.delete();
```

Comment:
Here it may fail, we'd better add the corresponding judgment, if it fails, print the corresponding log (if the failure result is not so important)